### ISSUE TITLE
remote access doesn't show display settings

### ISSUE THREAD

**Issue by Noelzu** [Ylianst/MeshCentral#5521]
**Describe the bug**
The display settings, that would normally be in the lower right corner, 
are not shown when connecting to a device with mutiple displays and you can only see all displays combined in one.
I get the same result no matter how I try to access the device and connect with it. (Columns, List, Desktops)

**To temporarily bypass the problem**
1. Go to a device
2. Select Desktop
3. Select Connect
4. Where the display settings would normally be, their is nothing
5. Switch to one of the tabs on the left side (My Device, My Account, My Events, My Files, My Users, My Server) 
(Just Disconnecting and Reconnecting doesn't work)
6. Access the tab from before again, now the display settings will be shown properly.
7. The next device you connect to has the same problem again.

**Expected behavior**
Display settings should be shown automatically as soon as you connect to a device with more than one display.

**Screenshots**
![lowertoolbar bug](https://github.com/Ylianst/MeshCentral/assets/149778788/0445a67e-8382-43c4-b072-5e8c8c7e2608)
![lowertoolbar bug 2](https://github.com/Ylianst/MeshCentral/assets/149778788/9e264e15-ff48-429b-ad20-4dba6a83a1cf)

**Server Software :**
 - OS: Debian 12 Bookworm
 - Virtualization: vSpher ESXi Hypervisor
 - Network: WAN, Apache Reverse Proxy
 - Version: 1.1.13
 - Node: 18.13.0

**Client Device:**
 - Device: Desktop PC
 - OS: Windows 10 Pro
 - Network: Remote over WAN
 - Browser: Google Chrome (118.0.0.0) (Edge (118.0.2088.76) and Firefox (116.0) also tested with the same result)

**Remote Device:**
 - Device: Desktop PC
 - OS: Windows 10 Pro
 - Network: Remote over WAN

**Comment by si458** [comment]
im not experiencing this issue?
first try, i see all displays and the icons without issues
![image](https://github.com/Ylianst/MeshCentral/assets/765314/468ae2a2-a351-46c7-8212-e21727c907c1)

**Comment by Noelzu** [comment]
my account does have full administrativ rights
![account](https://github.com/Ylianst/MeshCentral/assets/149778788/70606ec5-39de-428e-9b53-be427e544ab1)

**Comment by Noelzu** [comment]
Also checked our User Group and it has Full Rights to 
the device groups of the devices that we are testing.
The device groups themself also require you to be either a specific 
admin account or part of our User Group to acces them. 

This issue also isn't exclusive to my own user account, but has 
been effecting every user that we have tried this with so far.

**Comment by si458** [comment]
have you checked with a different remote device with multiple monitors if this is happen to it as well?

**Comment by Noelzu** [comment]
Sorry for the delayed response, the issue is sadly not an isolated case. 

It has consistenly appeared on every device with multiple monitors, that we have tested.

**Comment by si458** [comment]
is this still an issue? 
have you tried to update to 1.1.22 and use node 18 or above?

**Comment by si458** [comment]
Can u share ur config.json please? Hide secret info

**Comment by Noelzu** [comment]
here is our config.json
[config.json](https://github.com/Ylianst/MeshCentral/files/15058599/config.json)

**Comment by si458** [comment]
Can u share a screen recording of this happening? I still can't replicate this bug!

**Comment by Noelzu** [comment]
ok here is a screen recording showing both the error and the workaround we have been using
importent for that workaround is to not disconnect from the client, just switching 
out of the current connected client session (like for example going to My Devices, My Account, 
My Events, My Files, My Users, My Server) and then going back to the client
i censored a few areas like the client screen with black bars


https://github.com/Ylianst/MeshCentral/assets/149778788/fc5b7dd7-b9d1-4e30-970c-8dd711c91872

**Comment by si458** [comment]
huh thats weird? will try and replicate it but just having problems trying to replicate it.
if you are happy to provide myself access (even just only web ui access) then please email me details etc :)

**Comment by Noelzu** [comment]
Thank you for all your advice. We managed to fix the issue. 
Our MeshCentral Server was going over a reverse proxy, and Apache had the line 
"Header always append X-Frame-Options Deny." However, the MeshCentral server was already 
setting that header as "Header always append X-Frame-Options SAMEORIGIN." We removed 
"Header always append X-Frame-Options Deny," and now the buttons to switch displays appear & work as they should.
